BootloaderCorePkg/Tools/GenerateKeys.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
+## @ GenerateKeys.py
+#
+# This script is an reference to generate RSA private keys required by slimbootloader.
+# This should be one time key generation for a project based on usecase needs.and this would
+# be prerequiste to generate this before Slimboot build. These keys should be used for
+# various components signing in Slimboot
+#
+#
+# Copyright (c) 2020, Intel Corporation. All rights reserved.<BR>
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+##
+
+##
+# Import Modules
+#
+import os
+import sys
+import datetime
+import ntpath
+import argparse
+import binascii
+from ctypes import *
+from CommonUtility import *
+
+
+RSA_KEY_SIZE = {
+        "RSA2048"      : '2048',
+        "RSA3072"      : '3072',
+    }
+
+def signing_keys():
+    signing_keys_list = []
+    signing_keys_list.append ([
+      # Key ID                          | Key File Name start |
+      # ===========================================================
+        # MASTER_KEY_ID is used for signing Slimboot Key Hash Manifest (KEYH Component)
+        ("MASTER_KEY_ID",               "MasterTestKey"),
+
+        # CFGDATA_KEY_ID is used for signing external Config data blob)
+        ("CFGDATA_KEY_ID",              "ConfigTestKey"),
+
+        # FIRMWAREUPDATE_KEY_ID is used for signing capsule firmware update image)
+        ("FIRMWAREUPDATE_KEY_ID",       "FirmwareUpdateTestKey"),
+
+        # CONTAINER_KEY_ID is used for signing container header with mono signature
+        ("CONTAINER_KEY_ID",            "ContainerTestKey"),
+
+        # CONTAINER_COMP1_KEY_ID, CONTAINER_COMP2_KEY_ID is used for signing container components
+        # One can add multiple component keys as needed.
+        ("CONTAINER_COMP1_KEY_ID",      "ContainerComp1TestKey"),
+        ("CONTAINER_COMP2_KEY_ID",      "ContainerComp2TestKey"),
+        ])
+
+    return signing_keys_list
+
+# Generate RSA Key based on required key size
+def GenerateRsaKeys (openssl_path, key_dir, key_size):
+    key_list = signing_keys()
+    for item in key_list:
+        for Key_name, Key_file_name in item:
+            cmd = '%s genrsa -F4 -out %s/%s_RSA%s.pem %s' % (openssl_path, key_dir, Key_file_name, key_size, key_size)

```

Comment:
In keyname, can we add 'priv' suffix so as to easily identify that this is a private key, such as,
ConfigTestKey_RSA2048**_priv**.pem

Reply:
Will do.